cpu/stm32l0/periph/i2c.c

```diff
@@ -403,7 +403,7 @@ int i2c_write_regs(i2c_t dev, uint8_t address, uint8_t reg, const void *data, in
     }
 
     /* Check to see if the bus is busy */
-    while ((i2c->ISR & I2C_ISR_BUSY) & tick--) {
```

Comment:
this looks it was a pretty mean bug before.

Reply:
yes, i don't know why i didn't noticed it before since i'm using I2C in a significant way with a lot of devices :s